Restore.cmd

```diff
@@ -48,6 +47,14 @@ call "%NugetExe%" restore "%RoslynRoot%build\ToolsetPackages\dev15Willow.project
 echo Restoring packages: Roslyn SDK
 call "%NugetExe%" restore "%RoslynRoot%build\ToolsetPackages\roslynsdk.project.json" %NuGetAdditionalCommandLineArgs% || goto :RestoreFailed
 
+echo Locating MSBuild for Solution restore
+call "%RoslynRoot%SetDevCommandPrompt.cmd" || goto :RestoreFailed
+
+REM If we have an applocal copy of MSBuild, pass it to NuGet.  Otherwise, assume NuGet knows how to find it.
+if exist "%DevenvDir%\..\..\MSBuild\15.0\Bin\MSBuild.exe" (
+    set NuGetAdditionalCommandLineArgs=-verbosity quiet -configfile "%NuGetExeFolder%\nuget.config" -Project2ProjectTimeOut 1200 -msbuildpath "%DevenvDir%\..\..\MSBuild\15.0\Bin"
```

Comment:
I could probably change it to be `set NuGetAdditionalCommandLineArgs=%NuGetAdditionalCommandLineArgs% -msbuildpath "%DevenvDir%\..\..\MSBuild\15.0\Bin"`, since we are already setting the values beforehand.